src/main/java/mil/dds/anet/AnetApplication.java

```diff
@@ -204,9 +205,15 @@ public void run(AnetConfiguration configuration, Environment environment) {
 		ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
 		AnetEmailWorker emailWorker = new AnetEmailWorker(engine.getEmailDao(), configuration, scheduler);
 		FutureEngagementWorker futureWorker = new FutureEngagementWorker(engine.getReportDao());
+		ReportPublicationWorker reportPublicationWorker = new ReportPublicationWorker(engine.getReportDao(), configuration);
 		
+		//Check for any reports that need to be published every 1 minutes.
+		//And run once in 5 seconds from boot-up. (give the server time to boot up).
+		scheduler.scheduleAtFixedRate(reportPublicationWorker, 0, 1, TimeUnit.MINUTES);
```

Comment:
Shouldn't the initial delay be larger than `0`?

Reply:
I will make it 5.